slimmer.gemspec

```diff
@@ -17,12 +17,12 @@ Gem::Specification.new do |s|
   s.rubyforge_project = "slimmer"
 
   s.add_dependency 'nokogiri', '>= 1.5.0', '< 1.7.0'
-  s.add_dependency 'rack', '>= 1.3.5'
+  s.add_dependency 'rack', '~> 1.0'
   s.add_dependency 'plek', '>= 1.1.0'
   s.add_dependency 'json'
   s.add_dependency 'null_logger'
   s.add_dependency 'rest-client'
-  s.add_dependency 'activesupport'
+  s.add_dependency 'activesupport', '~> 4.0'
```

Comment:
I don't think you can pin this as we're currently testing support for Rails 3, 4 and 5 https://github.com/alphagov/slimmer/blob/master/jenkins.sh#L24-L40